CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
 - `[*]` Add Node 12 to CI ([#8411](https://github.com/facebook/jest/pull/8411))
 - `[*]` [**BREAKING**] Upgrade to Micromatch v4 ([#8852](https://github.com/facebook/jest/pull/8852))
 - `[babel-plugin-jest-hoist]` [**BREAKING**] Use ESM exports ([#8874](https://github.com/facebook/jest/pull/8874))
+- `[docs]` Add `collectCoverage` to CLI Reference ([#8996](https://github.com/facebook/jest/pull/8996))
```

Comment:
Changelog entry now outdated (PR no longer does quite that). Would also be fine for me to omit the changelog entry now that it's a very small doc change.

Reply:
```suggestion
- `[docs]` Add alias and optional boolean value to `coverage` CLI Reference ([#8996](https://github.com/facebook/jest/pull/8996))
```